extension.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "SemanticMediaWiki",
-	"version": "3.1.3",
+	"version": "3.1.3-alpha",
```

Comment:
@kghbln Could it be that you meant 3.1.4-alpha?

Reply:
Oops. Thanks for the pointer.